### ISSUE TITLE
adrama.to

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#4297]
### Basic Information

**Test link:** `http://www1.adrama.to/watch-online-white-war-episode-01-243519.html`
**Category:** ads
**Reported from:** United States
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.116 Safari/537.36
**Extension:** Nano Adblocker 1.0.0.150

### Additional Message

unblocked 3p crap
debugger trap

open debugger on this url: `http://www1.adrama.to/vidembed.T3GR9C-B3MfrABRKbj4Ssw.html` and player redirects to google (out of scope?)
